app/services/create_induction_tutor.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class CreateInductionTutor < BaseService
+  attr_accessor :school, :email, :full_name, :profile
+
+  def initialize(school:, email:, full_name:)
+    @school = school
+    @email = email
+    @full_name = full_name
+  end
+
+  def call
+    ic_profile = nil
+
+    ActiveRecord::Base.transaction do
+      school.induction_coordinators.first.destroy! if school.induction_coordinators.first
+
+      user = User.create!(full_name: full_name, email: email)
+      ic_profile = InductionCoordinatorProfile.create!(user: user, schools: [school])
+      SchoolMailer.nomination_confirmation_email(user: user, school: school, start_url: start_url).deliver_now
+    end
+
+    self.profile = ic_profile
```

Comment:
Why do we not want to set the profile inside the transaction?

Reply:
Probably could do that, I think I was thinking if the transaction rolled back it would leave the `profile` with and invalid value if it was assigning to `self` inside the block... unlikely to happen perhaps as it would only be if the mailer failed?
BTW, do we not have background workers? - was wondering why the mailer was using `deliver_now` instead of `deliver_later`?

Reply:
I mean, unlikely failures are precisely the reason we do stuff in transactions.

We have a worker, although we added it relatively recently. You can see the first ever usage of `deliver_later` on https://github.com/DFE-Digital/early-careers-framework/pull/392. 

I am pretty sure we are using `deliver_now` because a developer who since left the project found `deliver_later` annoying, with it increasing the time you would need to wait for your email if the worker was busy with something heavy at the time. Or at least I think that was the worry when we discussed using it half a year ago for sending sign-in emails.

Reply:
I'll make it `deliver_later` if there are issues with workers then that's an infrastructure thing.

Reply:
Oh, you are gonna love this. I think making it `deliver_later` means that it's not immediately available in the cypress tests (they use test mailer to figure out which emails were sent). Argh. Can you make it `deliver_now` and we can look into changing it later?

Reply:
Yes thanks, had a chat with @BroiSatse and we'll need to look at this later.